### ISSUE TITLE
Security Vulnerability

### ISSUE THREAD

**Issue by Retr02332** [pvorb/clone#120]
Hi !!

Do you have an email address where I can send a security vulnerability?

Regards.

**Comment by pvorb** [comment]
Yes, it's paul at vorba dot ch. I also have a PGP key at https://keybase.io/pvorb if you want to use that.

Thanks

**Comment by Retr02332** [comment]
Perfect, I sent you the information to this email

Regards.

**Comment by 1muen** [comment]
Is there a vulnerability? If there isnt it would be best to close the issue, if there is, it would be best, if there would be a patch + warning.
